db/database.go

```diff
@@ -917,7 +921,7 @@ func (db *Database) Compact() (int, error) {
 	}
 
 	// Trigger view compaction for all tombstoned documents older than the purge interval
-	purgeIntervalDuration := time.Duration(-db.PurgeInterval) * time.Hour
+	purgeIntervalDuration := -db.PurgeInterval * time.Hour
```

Comment:
`db.PurgeInterval` is already an absolute duration (in nanoseconds), I don't think we want to multiply by hour again when it gets used?

Reply:
Yeah, it's true, db.PurgeInterval is already an absolute duration (in nanoseconds) and we don't want to multiply by hour again when it gets used.